src/templates/about-page.js

```diff
@@ -56,33 +61,52 @@ export const AboutPageTemplate = ({
                 width: '80%',
                 marginLeft: '10%',
               }}>
-                <h2 className="is-size-3">{heading}</h2>
+                <h2 className="is-size-4">{headingAboutTechConnect}</h2>
                 <hr style={{
                   color:'#1C2833',
                   backgroundColor: '#1C2833',
                   height: 5
                 }}/>
-                <p >{description}</p>
+                <Row gutter={40}>
+                  <Col span={12}><p className="is-size-5">{aboutTechConnect}</p></Col>
+                  <Col span={12}><PreviewCompatibleImage imageInfo={image2} /></Col>
+                </Row>
               </div>
+              <div style={{
+                width: '80%',
+                marginLeft: '10%',
+                paddingTop: '5%'
+              }}>
+                <h2 className="is-size-4">{headingAboutMAF}</h2>
```

Comment:
I feel like this heading line should be abstracted into a component, since you're using it more than once...

Reply:
Which part are you referring to, exactly?

Reply:
the "h2" plus the "hr" sections into a reusable sectionHeader component that can be instantiated the way navbar and previewCompatibleImage are instantiated.